common/app/components/Player.tsx

```diff
@@ -962,16 +962,18 @@ const Player = React.memo(function Player({
     }, [clock, rewindSubtitle?.start, mediaAdapter, seek]);
 
     useEffect(() => {
-        if (!webSocketClient || extension.supportsWebSocketClient) {
-            // Do not handle mining commands here if the extension supports the web socket client.
-            // The extension will handle the commands for us.
+        // Return early if:
+        // 1. No WebSocket client, OR
+        // 2. Extension supports WebSocket clients AND this is NOT a local video (otherwise the extension will handle it)
+        const isLocalVideoFile = Boolean(videoFileUrl);
+        if (!webSocketClient || (extension.supportsWebSocketClient && !isLocalVideoFile)) {
```

Comment:
On reviewing the existing implementation and behavior the actual correct change here is to remove the `supportsWebSocketClient` check entirely. The extension is not forwarding "load-subtitle" nor "seek" commands to the asbplayer app, because there's no really good reason for it to, unlike "mine-subtitle."

So the check should be 
```
if (!webSocketClient) {
  return;
}
```